app/assets/stylesheets/static_pages.scss

```diff
@@ -39,4 +39,16 @@ div.std-margin-top {
 a.btn-sign-log {
   width: 100%;
   max-width: 150px;
+}
+
+li.add-padding {
+  padding: 0 10px 15px 0;
+}
+
+span.name {
+  font-size: 13px;
+}
+
+span.district {
+  font-size: 12px;
 }
```

Comment:
Is that a question or a statement. If it's a question, scoping styles per controller it's a bit tricky in rails, since all the css files go through the [asset pipeline](http://guides.rubyonrails.org/asset_pipeline.html), concatenated and minified into one large css.

If you want styles inside `playspaces.scss` to scope to the playspaces views, you should create a wrapper that can isolate the styles to add specificity. The best way i've found is to create a wrapper in the controller views like:

```html
<div id="playspaces">
   <h1> ...
</div>
```

and then in the `playspaces.scss`:

```css
#playspaces {
  h1 { color: red }
  ...
}
```

Let me know :)